coral/src/App.test.tsx

```diff
@@ -0,0 +1,12 @@
+import App from "./App"
+import { render, screen } from "@testing-library/react";
+
+
+describe("pp.tsx", ()=> {
+  it("shows a headline", ()=> {
+    render(<App />)
+    const heading = screen.getByRole("heading")
+
+    expect(heading).toHaveTextContent("Hello Klaw 👋")
```

Comment:
😄  I really like that you noticed that, because it's the better way to use the selector. 

I added it that way to confirm that `jest-dom` matcher is working but now I realize I should have used `.toBeVisible()` instead 🤦  Will update that!